airflow/docker-compose.yaml

```diff
@@ -0,0 +1,273 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Apache 2.0 license requires us to include attribution notes (NOTICE file) from the original library and to list changes we have made to comply with the license.

Reply:
We have not changed Airflow, though? And we don't plan to distribute Airflow. So I would say stricly the notice file might not be neccessary, but it certainly can't hurt and it is good to be mindful of license issues.
I am not the greatest expert on software licenses, but this TLDR seems to reflect Apache 2.0 quite well: https://tldrlegal.com/license/apache-license-2.0-(apache-2.0)

Reply:
Let's err on the side of caution, I would say.